src/main/java/com/mashape/unirest/http/options/Options.java

```diff
@@ -70,9 +73,23 @@ public static void refresh() {
 		PoolingHttpClientConnectionManager syncConnectionManager = new PoolingHttpClientConnectionManager();
 		syncConnectionManager.setMaxTotal((Integer) maxTotal);
 		syncConnectionManager.setDefaultMaxPerRoute((Integer) maxPerRoute);
-		
+
+		// Get retry handler if set
+		HttpRequestRetryHandler httpRequestRetryHandler = (HttpRequestRetryHandler) Options.getOption(Option.HTTP_REQUEST_RETRY_HANDLER);
+		if (httpRequestRetryHandler == null) {
+			httpRequestRetryHandler = Options.HTTP_REQUEST_RETRY_HANDLER;
```

Comment:
@praves77 looks like if we get `Options.HTTP_REQUEST_RETRY_HANDLER` and it's `null`, we will reassign the `null`. I did a little fix in the next PR